wrapanapi/systems/container/rhkubernetes.py

```diff
@@ -34,16 +34,14 @@
 """
 
 
-class Kubernetes(WrapanapiAPIBase):
+class Kubernetes(System):
 
     _stats_available = {
         'num_container': lambda self: len(self.list_container()),
         'num_pod': lambda self: len(self.list_container_group()),
         'num_service': lambda self: len(self.list_service()),
         'num_replication_controller':
             lambda self: len(self.list_replication_controller()),
-        'num_replication_controller_labels':
```

Comment:
Q: it seems it isn't used in tests. however, what is this change for ?

Reply:
`list_replication_controller_labels` is not an existing method in this class, I think my linter caught this and that is likely why I took this out.